pkg/logging/log.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+)
+
+func InfoLogging(output string) {
+	color.Blue("Info:")
+	fmt.Printf("%s\n", output)
+}
+
+func WarningLogging(output string) {
```

Comment:
Instead of the calling needing to call format on the args, they could be passed like this and then this method can format it.
```suggestion
func Warning(output string, args ...string) {
```

Reply:
It should be `interface` instead of `string`. In the rook code base I see similar helpers that make it nicer for logging.
```go
Infof(format string, args ...interface{})
Debugf(format string, args ...interface{})
Warningf(format string, args ...interface{})
```

Reply:
done